src/test/java/io/dashboardhub/pipelinedashboard/definitions/GuestDefinitions.java

```diff
@@ -21,6 +21,5 @@ public void I_try_to_access_a_secure_page() {
     @Then("^I get redirected to the login page$")
     public void I_get_redirected_to_the_login_page() {
         Assert.assertEquals(withBaseUrl("/login"), driver.getCurrentUrl());
-        driver.close();
```

Comment:
I have added it to this ticket, which I do shortly https://github.com/DashboardHub/PipelineDashboard/issues/195